### PR TITLE
Investigate how to use Prow for unify ci

### PR DESCRIPTION
**Pull request by JianLi-RH**
1. Adding one config file for ocp 4.7 upgrade, upgrade from releases:initial to "4.7.24"
ci-operator/config/openshift/cucushift/openshift-cucushift-master__unifyci-4.7.yaml
2. Adding test step and workflow
ci-operator/step-registry/openshift/unify-ci/cucushift

/cc @jhou1 @liangxia @dis016

### CODE REVIEW

File: ci-operator/step-registry/openshift/unify-ci/cucushift/upgrade/openshift-unify-ci-cucushift-upgrade-commands.sh [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=${KUBECONFIG}
+export KUBECONFIG=${KUBECONFIG}
```
Comment (jhou1) [inline ci-operator/step-registry/openshift/unify-ci/cucushift/upgrade/openshift-unify-ci-cucushift-upgrade-commands.sh:43]: `KUBECONFIG` is exported by default, https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables

**Comment by JianLi-RH** [comment]
/hold
We need to do more investigation on the code structure
/cc @jhou1 @liangxia @dis016 @pruan-rht

**Comment by liangxia** [comment]
> We need to do more investigation on the code structure

+1     Unify CI is our task. I think we need a better folder name here.

**Comment by liangxia** [comment]
@JianLi-RH   Really great progress. Thank you.

**Pull request closed without merging**